ztest/ztest.go

```diff
@@ -31,6 +31,50 @@
 //      COUNT
 //      2
 //
+// Alternatively, tests can be configured to run as shell scripts.
```

Comment:
We should add a note at the top of the comments (before the first description) saying that the yaml files are used for two different cases.

Reply:
Sounds good.